net/pdnsd/files/pdnsd.init

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2019 OpenWrt.org
+
+USE_PROCD=1
+
+START=65
+STOP=10
+
+PROG=/usr/sbin/pdnsd
+CONFFILE=/etc/config/pdnsd.conf
+PIDFILE=/var/run/pdnsd.pid
+CACHEDIR=/tmp/pdnsd
+CACHEFILE=$CACHEDIR/pdnsd.cache
+
+start_service() {
+	gen_cache
+	procd_open_instance
+	procd_set_param command "$PROG" -c "$CONFFILE" -p "$PIDFILE"
+	procd_set_param limits nofile=4096
+	procd_set_param stdout 1
+	procd_set_param stderr 1
```

Comment:
Consider patching pdnsd to log directly with syslog when evening running on the foreground.  This is not a requirement, but it's less runtime overhead and possibly the log lines are more readable from logread output.

Reply:
You mean remove them, right？

Reply:
Only remove them if you can make pdnsd doing `syslog()` while running in the foreground.

Reply:
keep it.